advanced-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# arg 1: old str, arg 2: new str
+replaceStrInNuxtConfigFile()
+{
+  cp example/nuxt.config.js example/staging.config.js
+  sed "s/$1/$2/g" < example/staging.config.js > example/nuxt.config.js
+  rm example/staging.config.js
+}
+
+yarn lint
+
+# Read GTM ID
+printf "GTM ID (empty for short test): "
+read GTM_ID
+
+if [ "$GTM_ID" = "" ] ; then
+  jest
+else
+  replaceStrInNuxtConfigFile "GTM-XXXXXXX" "$GTM_ID"
```

Comment:
We can simply modify nuxtConfig object in tests. Any special reason for this bash script?

Reply:
I hate to change variables in files, when it can be automatizate.
With this script, if you want to make tests with real GTM code, you just need to run the `npm run test` and insert your GTM code.

Reply:
What do you think?

Reply:
Maybe we can support and use environment variables? `GTM_ID=GTM-YYYY yarn dev`

Reply:
Ok. I will check and update it today evening or tomorrow morning. Thanks for review.

Reply:
Thanks :)